doc/static_discovery_configuration_guide.md

```diff
@@ -0,0 +1,808 @@
+            </ns5:PartyInfo>
+            <ns5:CollaborationInfo>
+               <ns5:Service type="tc1">bdx:noprocess</ns5:Service>
+               <ns5:Action>TC1Leg1</ns5:Action>
+               <ns5:ConversationId>bd3ea133-6a0d-11ec-9789-0af9f1d3371a@edelivery.digital</ns5:ConversationId>
+            </ns5:CollaborationInfo>
+            <ns5:MessageProperties>
+               <ns5:Property name="originalSender">urn:oasis:names:tc:ebcore:partyid-type:unregistered:C1</ns5:Property>
+               <ns5:Property name="finalRecipient">urn:oasis:names:tc:ebcore:partyid-type:unregistered:C4</ns5:Property>
+            </ns5:MessageProperties>
+            <ns5:PayloadInfo>
+               <ns5:PartInfo href="cid:message">
+                  <ns5:PartProperties>
+                     <ns5:Property name="MimeType">text/xml</ns5:Property>
+                  </ns5:PartProperties>
+               </ns5:PartInfo>
+            </ns5:PayloadInfo>
+         </ns5:UserMessage>
+      </ns5:Messaging>
+   </soap:Header>
+   <soap:Body>
+      <ns2:retrieveMessageResponse xmlns:ns2="http://org.ecodex.backend/1_1/" xmlns:ns3="http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/" xmlns:ns5="http://www.w3.org/2003/05/soap-envelope" xmlns:xmime="http://www.w3.org/2005/05/xmlmime">
+         <payload payloadId="cid:message">
+            <value>PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiPz4KPGhlbGxvPndvcmxkPC9oZWxsbz4=</value>
+         </payload>
+      </ns2:retrieveMessageResponse>
+   </soap:Body>
+</soap:Envelope>
+```
+
+The content inside the payload's `value` element is base64 encoded. After [decoding](https://www.base64decode.org/), the value looks like this:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<hello>world</hello>
+```
+
+If you want to try sending the same message from Access Point 2 (`org2_gw`) to Access Point 1 (`org1_gw`), just switch
+the values of the `From` and `To`, and `originalSender` and `finalRecipient` properties:
+
+```
+<ns:PartyInfo>
+   <ns:From>
+      <ns:PartyId type="urn:oasis:names:tc:ebcore:partyid-type:unregistered">harmony-org2</ns:PartyId>
+      <ns:Role>http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/initiator</ns:Role>
+   </ns:From>
+   <ns:To>
+      <ns:PartyId type="urn:oasis:names:tc:ebcore:partyid-type:unregistered">harmony-org1</ns:PartyId>
+      <ns:Role>http://docs.oasis-open.org/ebxml-msg/ebms/v3.0/ns/core/200704/responder</ns:Role>
+   </ns:To>
+</ns:PartyInfo>
+.
+.
+<ns:MessageProperties>
+   <ns:Property name="originalSender">urn:oasis:names:tc:ebcore:partyid-type:unregistered:C4</ns:Property>
+   <ns:Property name="finalRecipient">urn:oasis:names:tc:ebcore:partyid-type:unregistered:C1</ns:Property>
+</ns:MessageProperties>
+```
+
+Then, send the updated request to Access Point 2 (`org2_gw`) using the curl command below. The request (`submitRequestSwitched.xml`) can be downloaded [here](configuration_examples/static_discovery/submitRequestSwitched.xml).
```

Comment:
Raw link comment also applies here.

Reply:
I updated the link to point to the raw file.